backend/apis/clients/get-client.api.js

```diff
@@ -83,21 +83,33 @@ function getClientById(connection, clientId, cbk) {
       ) intake ON intake.clientId = clients.id
       INNER JOIN users created ON created.id = clients.addedBy
       INNER JOIN users modified ON modified.id = clients.modifiedBy
-      WHERE clients.id = ?
+      WHERE clients.id = ?;
+
+    SELECT serviceId, serviceName
```

Comment:
The `GET /clients/:id` endpoint was actually not returning the intake services correctly. So I first had to fix that.